.ci/ci-run-with-core.sh

```diff
@@ -8,4 +8,4 @@ export M2_HOME=/usr/share/maven
 # sudo apt-get update
 # sudo apt-get install -y xmlstarlet
 
-mvn clean install -B -f src/repairnator-core/ && mvn ${MAVEN_OPTS} -Dtest=$TEST_LIST clean test -B -f $TEST_PATH -Dmaven.resolver.transport=native  -Daether.connector.connectTimeout=300000 -Daether.connector.requestTimeout=300000
+mvn clean install -B -f src/repairnator-pipeline/ && mvn clean install -B -f src/repairnator-core/ && mvn ${MAVEN_OPTS} -Dtest=$TEST_LIST clean test -B -f $TEST_PATH -Dmaven.resolver.transport=native  -Daether.connector.connectTimeout=300000 -Daether.connector.requestTimeout=300000
```

Comment:
Hi @sofiabobadilla,

This should not work since you are trying to install `repairnator-pipeline` before `repairnator-core`.
What I was suggesting was to create a new script called `ci-run-with-core-pipeline.sh` that would run the command:

`mvn clean install -B -f src/repairnator-core/ && mvn clean install -B -f src/repairnator-pipeline/ && mvn ${MAVEN_OPTS} -Dtest=$TEST_LIST clean test -B -f $TEST_PATH -Dmaven.resolver.transport=native  -Daether.connector.connectTimeout=300000 -Daether.connector.requestTimeout=300000`

Then you would replace the name of the script in the `repairnator-realtime` step of the Jenkinsfile

Reply:
I see your point now, thanks André!

Reply:
It worked!! :D